control/ipv6.yaml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.3
+info:
+  title: Control ipv6 models
+  description: >-
+    All control ipv6 schemas
+  version: ^0.0.0
+components:
+  schemas:
+    Action.Protocol.Ipv6:
+      description: >-
+        Actions associated with IPv6 on configured resources.
+      type: object
+      required:
+        - choice
+      properties:
+        choice:
+          type: string
+          x-enum:
+            ping:
+              x-field-uid: 1
+          x-field-uid: 1
+        ping:
+          $ref: "#/components/schemas/Action.Protocol.Ipv6.Ping"
+          x-field-uid: 2
+    Action.Response.Protocol.Ipv6:
+      description: >-
+        Response for actions associated with IPv6 on configured resources.
+      type: object
+      required:
+        - choice
+      properties:
+        choice:
+          type: string
+          x-enum:
+            ping:
+              x-field-uid: 1
+          x-field-uid: 1
+        ping:
+          $ref: "#/components/schemas/Action.Response.Protocol.Ipv6.Ping"
+          x-field-uid: 2
+    Action.Protocol.Ipv6.Ping:
+      description: Request for initiating ping between multiple source and destination pairs.
+      type: object
+      properties:
+        requests:
+          description: >-
+            List of IPv6 ping requests.
+          type: array
+          items:
+            $ref: "#/components/schemas/Action.Protocol.Ipv6.PingRequest"
+          x-field-uid: 1
+    Action.Protocol.Ipv6.PingRequest:
+      description: >-
+        Request for initiating ping between a single source and destination pair.
+
+        For ping request, 1 IPv6 ICMP Echo Request shall be sent and wait for ping response to
+        either succeed or time out. The API wait timeout for each request shall be 300ms.
```

Comment:
Same question as earlier.  Why 300ms?

Reply:
answered in comment for ipv4 ping request